tools/cldr-apps/src/main/java/org/unicode/cldr/web/SurveyAjax.java

```diff
@@ -2861,6 +2880,9 @@ public static void getRow(HttpServletRequest request, HttpServletResponse respon
                 }
 
                 try {
+                    /*
+                     * TODO: why use org.json.JSONWriter here but SurveyAjax.JSONWriter elsewhere?
+                     */
```

Comment:
Comment outdated by https://github.com/unicode-org/cldr/pull/1002 -- I'll fix it later